ntc_templates/templates/arista_eos_show_interfaces_status.textfsm

```diff
@@ -4,11 +4,11 @@ Value STATUS (connected|notconnect|inactive|errdisabled|disabled)
 Value VLAN_ID (\S+)
 Value DUPLEX (\S+)
 Value SPEED (\S+)
-Value TYPE (\S.*)
+Value TYPE (\S.*\S)
 Value FLAGS (\S+)
 
 Start
-  ^${PORT}(?:\s+${NAME})?\s+${STATUS}\s+(?:in\s+)?${VLAN_ID}\s+${DUPLEX}\s+${SPEED}\s+${TYPE}(?:\s+${FLAGS})?\s*$$ -> Record
-  ^Port\s+Name\s+Status\s+Vlan\s+Duplex\s+Speed\s+Type(?:\s+Flags)?\s*$$
+  ^${PORT}(?:\s+${NAME})?\s+${STATUS}\s+(?:in\s+)?${VLAN_ID}\s+${DUPLEX}\s+${SPEED}\s+${TYPE}(?:\s+${FLAGS})?(?:\s+\S+)?\s*$$ -> Record
+  ^Port\s+Name\s+Status\s+Vlan\s+Duplex\s+Speed\s+Type(?:\s+Flags)?(?:\s+Encapsulation)?\s*$$
```

Comment:
Since no one actually has any values for Flags and Encapsulation data (and up to now, no one cares about it)--can we just clear the header row regardless of whether Flags and/or Encapsulation exist in the header row?

Reply:
> Since no one actually has any values for Flags and Encapsulation data (and up to now, no one cares about it)--can we just clear the header row regardless of whether Flags and/or Encapsulation exist in the header row?

Flags was optional from an earlier PR and Encapsulation is optional as of this PR.

I think we're good on the header row at this point, right?

Reply:
Header row I would just do:

`^Port\s+Name.*$$`

i.e. we are just clearing that line, why not make it simpler (but very unlikely to be missed). In other words, both the `Flags` and `Encapsulation` issue only came up only because of the line we are trying to clear.